run_infrastructure.py

```diff
@@ -91,6 +92,7 @@ def run_infrastructure(
             datastores,
             docker_compose_path=path,
             pytest_path=pytest_path,
+            analytics_opt_out=analytics_opt_out
```

Comment:
And a trailing `,` here... 🤷‍♂️ 
```suggestion
            analytics_opt_out=analytics_opt_out,
```

Reply:
Looks like the `,` is still missing. I'm asking you to add it in because auto-formatting tools will do so the next time this file is touched, and it's likely that the rest of the changes would be unrelated at that point.